<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>0</height>
       </size>
      </property>
      <row>
       <property name="text">
        <string>0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ZugKlasse</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Züge gesamt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NutzenWirkZüge</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nutzen Pro Jahr</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>NRz</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>FRz</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>S</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>Gesamt</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Enter the Difference Value in Secs</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Takt</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Zug. Nr.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Zug. Klasse</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Zug. Charak</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stations Index</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stationskürzel</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NullVariante</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximumSize">
       <size>
        <width>1194</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="ColumnNameSelectedForComparision">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>1194</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Update Nutzen Column(s) and Display Gesamtnutzen [pro Jahr]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionOpen_Folder_Location"/>
   </widget>
   <widget class="QMenu" name="menuAbout_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionUsage_Of_Application"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuAbout_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_Folder_Location">
   <property name="text">
    <string>Open Folder Location</string>
   </property>
  </action>
  <action name="actionUsage_Of_Application">
   <property name="text">
    <string>Usage_Of_Application</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
